host/xtest/Makefile

```diff
@@ -26,6 +26,9 @@ GP := _gp
 endif
 
 srcs := xtest_1000.c \
+	xtest_2000.c \
+	sock_server.c \
+	rand_stream.c \
```

Comment:
Since GP sockets support is optional in tee-supplicant (and I'd like it to be optional too in optee_os), this should depend on the `CFG_` variable as well.